lib/printService.js

```diff
@@ -30,24 +31,24 @@ class PrintService {
   }
 
   async printReservations(printGroupId, bookingId, reservationIds, opts = {}) {
-    const tickets = await new QuickTravelApi(this.quicktravel_url, this.csrfToken)
+    const tickets = await new QuickTravelApi(this.quicktravel_url, this.csrfToken, this.qt_print_server_type)
```

Comment:
I almost wonder if there's some way to do this in the constructor, e.g.:
```
this.api = new QuickTravelApi(this.quicktravel_url, this.csrfToken, this.qt_print_server_type)
```

Then in these methods we could just say
```
const tickets = await this.api.issueAndPrint(bookingId, reservationIds, opts);
```

I just feel setting up this api object is a bit repetitive.